paper-typeahead-input.html

```diff
@@ -233,6 +233,21 @@
         notify:true
       },
       /**
+       * A boolean property to tell if need to open show up with empty input string.
+       * isAllowSuggestOnEmptyInput
+       */
+      isAllowSuggestOnEmptyInput: {
+        type:Boolean,
+        value:false
+      },
+      /**
+       * A boolean property to tell if need to open show up on focusing input element.
+       */
+      suggestOnFocus: {
+        type:Boolean,
```

Comment:
Both twitter typeahead.js and google.com search has this suggest on focus and clear suggest when lose focus, so I think maybe it's good to have the default value to be true to encourage developer to be consistent with common user experience.